app/controllers/events_controller.rb

```diff
@@ -22,6 +23,7 @@ def edit
   # POST /events
   def create
     @event = Event.new(event_params)
+    @event.creator = current_user
 
     if @event.save
       @event.editors << current_user
```

Comment:
Note: Another team already added the editors collection to the event, maybe every editor should be allowed to edit the event, but only creates should be allowed to mange the editors.